scripts-available/CDB_CartodbfyTable.sql

```diff
@@ -897,7 +898,22 @@ BEGIN
   END IF;
 
   -- We must rewrite, so here we go...
-
+  
+  -- Our desired PK sequence name
+
+  -- We are going to drop the source table when we're done anyways
+  -- but it's possible the source PK sequence is living in a name we would like to use
+  -- so we check to see if that's the case, and rename it out of the way
+  IF has_usable_primary_key AND has_usable_pk_sequence THEN
+    -- See if the existing sequence is squatting on our preferred name
+    destseq := Format('%s_%s_seq', relname, const.pkey);
+    SELECT pg_catalog.pg_get_serial_sequence(Format('%I.%I', relschema, relname), const.pkey)
+      INTO relseq;
+    -- If it's the name we want, then rename it
+    IF relseq IS NOT NULL AND relseq = Format('%I.%I', destschema, destseq) THEN
+      PERFORM _CDB_SQL(Format('ALTER SEQUENCE %s RENAME TO %s_tmp', relseq, destseq), '_CDB_Rewrite_Table');
```

Comment:
I must be missing something: it seems to me it's renaming the existing seq _relseq_ to the desired _destseq_ (<relname>_cartodb_id_seq) when it already has that very same name (?)

Reply:
It's moving `%s_cartodb_id_seq` to `_cartodb_id_seq_tmp` so that in the following steps, when the new table is created, it will be possible for the serial in `cartodb_id` to use `%s_cartodb_id_seq` as it's sequence name. Probably it would be best to recognize this case (sequence already there, and usable, even if the table is going to be re-written) as another case, but that would require larger changes below, to address what is a fairly minor corner case right now (the `ogr2ogr` upload issue).

Reply:
For some reason I was missing the `_tmp` 😅  thanks for the clarification!